src/components/GoogleAuth/loginComponent.js

```diff
@@ -35,6 +35,11 @@ const IndexPage = ({ classes }) => {
 
   return (
     <>
+      {/* eslint-disable no-console */}
+      {console.group('%cThe following data is required by the auth system', 'background:aquamarine; color:#fff;')}
```

Comment:
@ogwuazors Can you remove console logs if not needed?

Reply:
done